utilities/audio_utils.py

```
@@ -134,13 +134,21 @@ def init_latent_cache() -> None:
                 # .wav files - compute latents from audio
                 speaker_wav_path = files['.wav']
                 logger.info(f"Processing .wav file: {speaker_wav_path}")
-                speaker_embedding = asyncio.run(process_speaker_audio(str(speaker_wav_path), enable_disk_cache=True))                                       
+                coro = process_speaker_audio(str(speaker_wav_path), enable_disk_cache=True)
+                # If an event loop is running, schedule task; otherwise run synchronously
+                try:
+                    loop = asyncio.get_running_loop()
+                except RuntimeError:
+                    loop = None
+                if loop and loop.is_running():
+                    asyncio.create_task(coro)
```

Ruff (0.14.1)

Store a reference to the return value of `asyncio.create_task`

(RUF006)